src/ewatercycle/base/forcing.py

```diff
@@ -123,107 +188,209 @@ def load(cls, directory: str | Path):
+                end_time='2001-01-01T00:00:00Z',
+                directory=PosixPath('/home/verhoes/git/eWaterCycle/ewatercycle/esmvaltool_output/tmp05upitxoewcrep_20230815_154640/work/diagnostic/script'),
+                shape=PosixPath('/home/verhoes/git/eWaterCycle/ewatercycle/src/ewatercycle/testing/data/Rhine/Rhine.shp'),
+                pr='OBS6_ERA5_reanaly_1_day_pr_2000-2001.nc',
+                tas='OBS6_ERA5_reanaly_1_day_tas_2000-2001.nc',
+                tasmin='OBS6_ERA5_reanaly_1_day_tasmin_2000-2001.nc',
+                tasmax='OBS6_ERA5_reanaly_1_day_tasmax_2000-2001.nc'
+            )
 
-    During run the recipe.path is overwritten with a temporary file containing the updated recipe.
+        To generate forcing from CMIP6 for the Rhine catchment for 2000-2001:
 
-    Args:
-        recipe: ESMValTool recipe
-        output_dir: Directory where output should be written to.
-            If None then output is written to generated timestamped directory.
+        .. code-block:: python
+
+            from pathlib import Path
+            from rich import print
+            from ewatercycle.base.forcing import GenericDistributedForcing
+
+            shape = Path("./src/ewatercycle/testing/data/Rhine/Rhine.shp")
+            cmip_dataset = {
+                "dataset": "EC-Earth3",
+                "project": "CMIP6",
+                "grid": "gr",
+                "exp": ["historical",],
+                "ensemble": "r6i1p1f1",
+            }
+
+            forcing = GenericDistributedForcing.generate(
+                dataset=cmip_dataset,
+                start_time="2000-01-01T00:00:00Z",
+                end_time="2001-01-01T00:00:00Z",
+                shape=shape.absolute(),
+            )
+            print(forcing)
 
-    Returns:
-        ESMValTool recipe output
+        Gives something like:
+
+        .. code-block:: python
+
+            GenericDistributedForcing(
+                start_time='2000-01-01T00:00:00Z',
+                end_time='2001-01-01T00:00:00Z',
+                directory=PosixPath('/home/verhoes/git/eWaterCycle/ewatercycle/esmvaltool_output/ewcrep0ibzlds__20230904_082748/work/diagnostic/script'),
+                shape=PosixPath('/home/verhoes/git/eWaterCycle/ewatercycle/src/ewatercycle/testing/data/Rhine/Rhine.shp'),
+                pr='CMIP6_EC-Earth3_day_historical_r6i1p1f1_pr_gr_2000-2001.nc',
+                tas='CMIP6_EC-Earth3_day_historical_r6i1p1f1_tas_gr_2000-2001.nc',
+                tasmin='CMIP6_EC-Earth3_day_historical_r6i1p1f1_tasmin_gr_2000-2001.nc',
+                tasmax='CMIP6_EC-Earth3_day_historical_r6i1p1f1_tasmax_gr_2000-2001.nc'
+            )
+
+        To retrieve data from ESFG you will need a ~/.esmvaltool/config-user.yml file with:
+
+        .. code-block:: yaml
+
+            search_esgf: when_missing
+            download_dir: ~/climate_data
+            rootpath:
+                CMIP6: ~/climate_data/CMIP6
+            drs:
+                CMIP6: BADC
```

Comment:
```suggestion
                CMIP6: ESGF
```
Follow standard ESGF by default unless you're on a site-specific machine (as documented here: https://docs.esmvaltool.org/projects/ESMValCore/en/latest/quickstart/configure.html#user-configuration-file)

Reply:
done